portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,18 +30,22 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.sps.data.Comment;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
 
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
   private ArrayList<Comment> comments = new ArrayList<>();
+  private ArrayList<String> mycomments = new ArrayList<>();
```

Comment:
You can just assign this into a `List` to keep it more generic.

`private List<String> myComments = new ArrayList<>();`